### ISSUE TITLE
chatFileFormatting

### ISSUE THREAD

**Issue by 3l1zzy** [3l1zzy/CommNetProj2#4]
figure out how to heck to get the chatfile formatting to work

- txt files have no pointers or seek, will have to either add some type of key character or figure out how to have it clip to the end

**Comment by 3l1zzy** [comment]
Things to adjust - I looked at my rubric to see exactly what the requirements were again:
- When a connection is made (a client connects to the server), the thread that manages the connection waits for a username, and then it sends the content of the chat file to the client.  
- the log should not start when the last user joins only

**Comment by 3l1zzy** [comment]
fixed out of place message prompt

todo:
- When a connection is made (a client connects to the server), the thread that manages the connection waits for a username, and then it sends the content of the chat file to the client.
- the log should not start when the last user joins only

**Comment by 3l1zzy** [comment]
fixed the late log start by moving the filewriter into the "only create one if there isn't one already" block in the server, which I probably should have caught a little earlier than I did. I also added a quick "before you give me any info aside from your username, here's what you missed" block in the server program and a "listen for any input" block in the chat client so it actually prints the backlog on its console. Attached are the consoles from both clients and the server.
[backlog.txt](https://github.com/3l1zzy/CommNetProj2/files/5434030/backlog.txt)


todo:
- When a connection is made (a client connects to the server), the thread that manages the connection waits for a username, and then it sends the content of the chat file to the client.
- the log should not start when the last user joins only
